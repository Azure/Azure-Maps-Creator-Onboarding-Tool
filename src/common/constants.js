export const PATHS = {
  INDEX: '/',
  VIEW_CONVERSIONS: '/view-conversions',
  CREATE_UPLOAD: '/create-upload',
  PROCESSING: '/processing',
  CREATE_GEOREFERENCE: '/create-georeference',
  REVIEW_CREATE: '/review-create',
  CONVERSION: '/review-create/conversion',
  IMDF_CONVERSION: '/review-create/imdf-conversion',
  PAST_CONVERSION: '/past-conversion',
  CONVERSIONS: '/conversions',
  LAYERS: '/layers',
  LEVELS: '/levels',
  INVALID_PATH: '/*',
};

export const ROUTE_NAME_BY_PATH = {
  [PATHS.INDEX]: 'home',
  [PATHS.CREATE_UPLOAD]: 'Create',
  [PATHS.VIEW_CONVERSIONS]: 'View',
  [PATHS.PROCESSING]: 'processing',
  [PATHS.CREATE_GEOREFERENCE]: 'prepare.drawing.package',
  [PATHS.LAYERS]: 'prepare.drawing.package',
  [PATHS.LEVELS]: 'prepare.drawing.package',
  [PATHS.INVALID_PATH]: 'redirect',
  [PATHS.REVIEW_CREATE]: 'prepare.drawing.package',
  [PATHS.CONVERSION]: 'create.indoor.map',
  [PATHS.IMDF_CONVERSION]: 'convert.indoor.map',
  [PATHS.CONVERSIONS]: 'All conversions',
};

export const PLACES_ROUTE_NAME_BY_PATH = {
  [PATHS.CREATE_GEOREFERENCE]: 'prepare.building.configuration',
  [PATHS.LAYERS]: 'prepare.building.configuration',
  [PATHS.LEVELS]: 'prepare.building.configuration',
  [PATHS.REVIEW_CREATE]: 'prepare.building.configuration',
};

export const TRUNCATE_FRACTION_DIGITS = 8;

export const HTTP_STATUS_CODE = {
  ACCEPTED: 202,
  OK: 200,
  BAD_REQUEST: 400,
  UNAUTHORIZED: 401,
  FORBIDDEN: 403,
  TOO_MANY_REQUESTS: 429,
};

export const isStagingEnv = process.env.REACT_APP_STAGING_ENV === 'true';

export const PLACES_PREVIEW = {
  DEFAULT_LANGUAGE: 'en',
  VERSION: 'places-1.0',
  DESCRIPTION: 'Converted by AzMapsCreatorOnboardingTool',
  STORAGE_RETENTION: 'day',
  SEARCH_PARAMETER: 'placespreview',
};
