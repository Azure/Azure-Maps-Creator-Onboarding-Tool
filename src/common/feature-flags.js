const flags = {
  onboardingEnabled: false,
};

export default flags;