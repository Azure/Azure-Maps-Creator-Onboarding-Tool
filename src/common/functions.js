/**
 * Given a single path name, returns a list of all path names from root to the given path name.
 * Ex. Given '/create/hello', returns ['/', '/create', '/create/hello']
 * @param {String} pathName the path name of the URL
 * @returns {String[]} an array of path names
 */
export const getSplitPaths = (pathName) => {
  if (typeof pathName !== 'string' || pathName === '') { return []; }
  if (pathName === '/') { return ['/']; }

  const pathNameLowerCase = pathName.toLowerCase();
  const splits = pathNameLowerCase.split('/');
  const splitPaths = [];
  for (let i = splits.length - 1; i > 0; i--) {
    let stringBuilder = '';
    for (let j = 1; j <= i; j++) {
      stringBuilder += '/' + splits[j];
    }
    splitPaths.push(stringBuilder);
  }
  splitPaths.push('/');
  splitPaths.reverse();

  return splitPaths;
};