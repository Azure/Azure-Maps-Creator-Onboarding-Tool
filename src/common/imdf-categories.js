export const imdfCategories = [
  'auditorium',
  'brick',
  'classroom',
  'column',
  'concrete',
  'conferenceroom',
  'drywall',
  'elevator',
  'escalator',
  'fieldofplay',
  'firstaid',
  'fitnessroom',
  'foodservice',
  'footbridge',
  'glass',
  'huddleroom',
  'kitchen',
  'laboratory',
  'library',
  'lobby',
  'lounge',
  'mailroom',
  'mothersroom',
  'movietheater',
  'movingwalkway',
  'nonpublic',
  'office',
  'opentobelow',
  'parking',
  'phoneroom',
  'platform',
  'privatelounge',
  'ramp',
  'recreation',
  'restroom',
  'restroom.family',
  'restroom.female',
  'restroom.female.wheelchair',
  'restroom.male',
  'restroom.male.wheelchair',
  'restroom.transgender',
  'restroom.transgender.wheelchair',
  'restroom.unisex',
  'restroom.unisex.wheelchair',
  'restroom.wheelchair',
  'road',
  'room',
  'serverroom',
  'shower',
  'smokingarea',
  'stairs',
  'steps',
  'storage',
  'structure',
  'terrace',
  'theater',
  'unenclosedarea',
  'unspecified',
  'vegetation',
  'waitingroom',
  'walkway',
  'walkway.island',
  'wood',
];
