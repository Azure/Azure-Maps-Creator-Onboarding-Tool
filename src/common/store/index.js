export { conversionSteps, useConversionStore } from './conversion.store';
export { useDissolvedExterior, useGeometryStore } from './geometry.store';
export { useLayersStore } from './layers.store';
export { useLevelsStore } from './levels.store';
export {
  progressBarStepsByKey,
  useCompletedSteps,
  useProgressBarSteps,
  useProgressBarStore,
} from './progress-bar-steps';
export { resetStores } from './reset';
export { LRO_STATUS, useResponseStore } from './response.store';
export { useReviewManifestJson, useReviewManifestStore } from './review-manifest.store';
export { getDomain, useUserStore } from './user.store';
