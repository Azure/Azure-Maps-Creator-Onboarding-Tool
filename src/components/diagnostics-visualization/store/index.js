import { applyMiddleware, createStore } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension';

import rootReducer from '../reducers';

// Middlewares go here
const middleware = {};

const configureStore = (initialState) => createStore(
  rootReducer,
  initialState,
  // Redux Dev Tools Middleware
  composeWithDevTools(applyMiddleware(...middleware)),
);

// pass an optional param to rehydrate state on app start
const store = configureStore();

// export store singleton instance
export default store;
