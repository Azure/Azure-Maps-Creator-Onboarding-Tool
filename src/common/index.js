import { PATHS, ROUTE_NAME_BY_PATH } from './constants';

export { PATHS, ROUTE_NAME_BY_PATH };