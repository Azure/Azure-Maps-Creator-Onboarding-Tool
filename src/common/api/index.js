// The following API is designed to be used only by the onboarding tool and is not supported for any other use.
import { getEnvs } from 'common/functions';
import { useUserStore } from '../store/user.store';

export const uploadFile = (file) => {
  const { geography, subscriptionKey } = useUserStore.getState();
  const url = `${getEnvs()[geography].URL}/manifest?api-version=2.0&subscription-key=${subscriptionKey}`;

  return fetch(url, {
    method: 'POST',
    headers: {
      'Content-Type': 'application/zip',
    },
    body: file,
  });
};

export const fetchFromLocation = (location) => {
  const { subscriptionKey } = useUserStore.getState();
  return fetch(`${location}&subscription-key=${subscriptionKey}`, {
    method: 'GET',
  });
};

export const deleteFromLocation = (fetchUrl) => {
  const { subscriptionKey } = useUserStore.getState();
  fetch(`${fetchUrl}&subscription-key=${subscriptionKey}`, {
    method: 'DELETE',
  });
};

export const fetchAddress = (address) => {
  const { geography, subscriptionKey } = useUserStore.getState();
  return fetch(`${getEnvs()[geography].URL}/search/address/json?subscription-key=${subscriptionKey}&api-version=1.0&query=${address}&limit=1`)
    .then((res) => res.json());
};