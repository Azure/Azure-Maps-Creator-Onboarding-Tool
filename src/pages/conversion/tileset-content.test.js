import { screen, render, fireEvent } from '@testing-library/react';

import TilesetContent from './tileset-content';
import { useConversionStore } from 'common/store';

jest.mock('./tileset-map', () => () => 'Tileset map here');

describe('tileset content', () => {
  beforeEach(() => {
    useConversionStore.setState({
      selectedStep: 0,
      tilesetOperationLog: null,
      tilesetOperationId: null,
      tilesetId: null,
    });
  });

  it('should render nothing by default', () => {
    const view = render(<TilesetContent />);
    expect(view).toMatchSnapshot();
  });

  it('should render tileset content', () => {
    useConversionStore.setState({
      selectedStep: 3,
      tilesetOperationLog: JSON.stringify({ foo: 'tileset-operation-log', baz: 'blee blue blah blueberry dee'}, null, 4),
      tilesetOperationId: 'tilset-operation-zxcvzxcv-asdf-dgfh',
      tilesetId: 17,
    });
    const view = render(<TilesetContent />);
    expect(view).toMatchSnapshot();
  });

  it('should render tileset content logs tab', () => {
    useConversionStore.setState({
      selectedStep: 3,
      tilesetOperationLog: JSON.stringify({ foo: 'hop-hei-la-la-lei', baz: 'blee blue blah blueberry dee'}, null, 4),
      tilesetOperationId: 'qqwer-asdf-zxcvzxcv-asdf-dgfh',
      tilesetId: 44,
    });
    const view = render(<TilesetContent />);
    const logsTab = screen.getByText('logs');
    fireEvent.click(logsTab);
    expect(view).toMatchSnapshot();
  });
});