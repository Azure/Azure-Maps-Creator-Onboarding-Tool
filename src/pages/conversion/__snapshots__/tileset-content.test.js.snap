// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tileset content should render nothing by default 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div />
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`tileset content should render tileset content 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  height: 100%;
}

.emotion-3 {
  overflow-x: auto;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        <div
          class="emotion-0"
        >
          <div
            class="ms-FocusZone emotion-2 ms-Pivot root-109"
            data-focuszone-id="FocusZone1"
            role="tablist"
          >
            <button
              aria-selected="true"
              class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link is-selected linkIsSelected-119"
              data-content="logs"
              data-is-focusable="true"
              data-title="logs"
              id="Pivot0-Tab0"
              name="logs"
              role="tab"
              tabindex="0"
              type="button"
            >
              <span
                class="ms-Button-flexContainer flexContainer-120"
                data-automationid="splitbuttonprimary"
              >
                <span
                  class="ms-Pivot-linkContent linkContent-115"
                >
                  <span
                    class="ms-Pivot-text text-116"
                  >
                     
                    logs
                  </span>
                </span>
              </span>
            </button>
            <button
              aria-selected="false"
              class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link link-127"
              data-content="rendered.map"
              data-is-focusable="true"
              data-title="rendered.map"
              id="Pivot0-Tab1"
              name="rendered.map"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="ms-Button-flexContainer flexContainer-120"
                data-automationid="splitbuttonprimary"
              >
                <span
                  class="ms-Pivot-linkContent linkContent-115"
                >
                  <span
                    class="ms-Pivot-text text-116"
                  >
                     
                    rendered.map
                  </span>
                </span>
              </span>
            </button>
          </div>
          <div
            aria-hidden="false"
            aria-labelledby="Pivot0-Tab0"
            class="itemContainer-110"
            role="tabpanel"
          >
            <div>
              <div>
                <h3>
                  meta.data
                </h3>
                <div>
                  operationId: 
                  tilset-operation-zxcvzxcv-asdf-dgfh
                </div>
                <div>
                  tilesetId: 
                  17
                </div>
                <div>
                  mapConfigurationId: 
                  N/A
                </div>
              </div>
              <div>
                <h3>
                  operation.log
                </h3>
                <pre
                  class="emotion-3"
                >
                  {
    "foo": "tileset-operation-log",
    "baz": "blee blue blah blueberry dee"
}
                </pre>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  height: 100%;
}

.emotion-3 {
  overflow-x: auto;
}

<div>
    <div
      class="emotion-0"
    >
      <div
        class="emotion-0"
      >
        <div
          class="ms-FocusZone emotion-2 ms-Pivot root-109"
          data-focuszone-id="FocusZone1"
          role="tablist"
        >
          <button
            aria-selected="true"
            class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link is-selected linkIsSelected-119"
            data-content="logs"
            data-is-focusable="true"
            data-title="logs"
            id="Pivot0-Tab0"
            name="logs"
            role="tab"
            tabindex="0"
            type="button"
          >
            <span
              class="ms-Button-flexContainer flexContainer-120"
              data-automationid="splitbuttonprimary"
            >
              <span
                class="ms-Pivot-linkContent linkContent-115"
              >
                <span
                  class="ms-Pivot-text text-116"
                >
                   
                  logs
                </span>
              </span>
            </span>
          </button>
          <button
            aria-selected="false"
            class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link link-127"
            data-content="rendered.map"
            data-is-focusable="true"
            data-title="rendered.map"
            id="Pivot0-Tab1"
            name="rendered.map"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="ms-Button-flexContainer flexContainer-120"
              data-automationid="splitbuttonprimary"
            >
              <span
                class="ms-Pivot-linkContent linkContent-115"
              >
                <span
                  class="ms-Pivot-text text-116"
                >
                   
                  rendered.map
                </span>
              </span>
            </span>
          </button>
        </div>
        <div
          aria-hidden="false"
          aria-labelledby="Pivot0-Tab0"
          class="itemContainer-110"
          role="tabpanel"
        >
          <div>
            <div>
              <h3>
                meta.data
              </h3>
              <div>
                operationId: 
                tilset-operation-zxcvzxcv-asdf-dgfh
              </div>
              <div>
                tilesetId: 
                17
              </div>
              <div>
                mapConfigurationId: 
                N/A
              </div>
            </div>
            <div>
              <h3>
                operation.log
              </h3>
              <pre
                class="emotion-3"
              >
                {
    "foo": "tileset-operation-log",
    "baz": "blee blue blah blueberry dee"
}
              </pre>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`tileset content should render tileset content logs tab 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  height: 100%;
}

.emotion-3 {
  overflow-x: auto;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        <div
          class="emotion-0"
        >
          <div
            class="ms-FocusZone emotion-2 ms-Pivot root-109"
            data-focuszone-id="FocusZone9"
            role="tablist"
          >
            <button
              aria-selected="true"
              class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link is-selected linkIsSelected-119"
              data-content="logs"
              data-is-focusable="true"
              data-title="logs"
              id="Pivot8-Tab0"
              name="logs"
              role="tab"
              tabindex="0"
              type="button"
            >
              <span
                class="ms-Button-flexContainer flexContainer-120"
                data-automationid="splitbuttonprimary"
              >
                <span
                  class="ms-Pivot-linkContent linkContent-115"
                >
                  <span
                    class="ms-Pivot-text text-116"
                  >
                     
                    logs
                  </span>
                </span>
              </span>
            </button>
            <button
              aria-selected="false"
              class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link link-127"
              data-content="rendered.map"
              data-is-focusable="true"
              data-title="rendered.map"
              id="Pivot8-Tab1"
              name="rendered.map"
              role="tab"
              tabindex="-1"
              type="button"
            >
              <span
                class="ms-Button-flexContainer flexContainer-120"
                data-automationid="splitbuttonprimary"
              >
                <span
                  class="ms-Pivot-linkContent linkContent-115"
                >
                  <span
                    class="ms-Pivot-text text-116"
                  >
                     
                    rendered.map
                  </span>
                </span>
              </span>
            </button>
          </div>
          <div
            aria-hidden="false"
            aria-labelledby="Pivot8-Tab0"
            class="itemContainer-110"
            role="tabpanel"
          >
            <div>
              <div>
                <h3>
                  meta.data
                </h3>
                <div>
                  operationId: 
                  qqwer-asdf-zxcvzxcv-asdf-dgfh
                </div>
                <div>
                  tilesetId: 
                  44
                </div>
                <div>
                  mapConfigurationId: 
                  N/A
                </div>
              </div>
              <div>
                <h3>
                  operation.log
                </h3>
                <pre
                  class="emotion-3"
                >
                  {
    "foo": "hop-hei-la-la-lei",
    "baz": "blee blue blah blueberry dee"
}
                </pre>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  height: 100%;
}

.emotion-3 {
  overflow-x: auto;
}

<div>
    <div
      class="emotion-0"
    >
      <div
        class="emotion-0"
      >
        <div
          class="ms-FocusZone emotion-2 ms-Pivot root-109"
          data-focuszone-id="FocusZone9"
          role="tablist"
        >
          <button
            aria-selected="true"
            class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link is-selected linkIsSelected-119"
            data-content="logs"
            data-is-focusable="true"
            data-title="logs"
            id="Pivot8-Tab0"
            name="logs"
            role="tab"
            tabindex="0"
            type="button"
          >
            <span
              class="ms-Button-flexContainer flexContainer-120"
              data-automationid="splitbuttonprimary"
            >
              <span
                class="ms-Pivot-linkContent linkContent-115"
              >
                <span
                  class="ms-Pivot-text text-116"
                >
                   
                  logs
                </span>
              </span>
            </span>
          </button>
          <button
            aria-selected="false"
            class="ms-Button ms-Button--action ms-Button--command ms-Pivot-link link-127"
            data-content="rendered.map"
            data-is-focusable="true"
            data-title="rendered.map"
            id="Pivot8-Tab1"
            name="rendered.map"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="ms-Button-flexContainer flexContainer-120"
              data-automationid="splitbuttonprimary"
            >
              <span
                class="ms-Pivot-linkContent linkContent-115"
              >
                <span
                  class="ms-Pivot-text text-116"
                >
                   
                  rendered.map
                </span>
              </span>
            </span>
          </button>
        </div>
        <div
          aria-hidden="false"
          aria-labelledby="Pivot8-Tab0"
          class="itemContainer-110"
          role="tabpanel"
        >
          <div>
            <div>
              <h3>
                meta.data
              </h3>
              <div>
                operationId: 
                qqwer-asdf-zxcvzxcv-asdf-dgfh
              </div>
              <div>
                tilesetId: 
                44
              </div>
              <div>
                mapConfigurationId: 
                N/A
              </div>
            </div>
            <div>
              <h3>
                operation.log
              </h3>
              <pre
                class="emotion-3"
              >
                {
    "foo": "hop-hei-la-la-lei",
    "baz": "blee blue blah blueberry dee"
}
              </pre>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
